projects/1.21/assets/data-essence/datanessence/lang/zh_cn.json

```
@@ -0,0 +1,686 @@
+  "item.minecraft.lingering_potion.effect.gender_euphoria": "滞留型性别亢奋药水",
+  "item.minecraft.lingering_potion.effect.long_gender_euphoria": "滞留型性别亢奋药水",
+  "item.minecraft.tipped_arrow.effect.gender_euphoria": "性别亢奋之箭",
+  "item.minecraft.tipped_arrow.effect.long_gender_euphoria": "性别亢奋之箭",
+
+  "gui.widget.fluid":  "%3$s：%1$s/%2$smB",
+  "gui.widget.fluid.without_max": "%2$s：%1$s",
+
+  "gui.essence_bar.essence_with_max": "工业源质：%1$s/%2$s",
+  "gui.essence_bar.lunar_essence_with_max": "月辉源质：%1$s/%2$s",
+  "gui.essence_bar.natural_essence_with_max": "自然源质：%1$s/%2$s",
+  "gui.essence_bar.exotic_essence_with_max": "奇异源质：%1$s/%2$s",
+
+  "gui.essence_bar.essence": "工业源质：%1$s",
+  "gui.essence_bar.lunar_essence": "月辉源质：%1$s",
+  "gui.essence_bar.natural_essence": "自然源质：%1$s",
+  "gui.essence_bar.exotic_essence": "奇异源质：%1$s",
+  "gui.essence_bar.unknown": "???：%1$s",
+
+  "data_tablet.critical_data_unlocked": "解锁关键数据",
+  "data_tablet.tier_upgrade": "你已晋升至",
+  "data_tablet.tier": "%1$s级",
+  "data_tablet.databank_minigame_completed": "已完成",
+  "data_tablet.databank_no_entries_left": "许可不足，无法显示更多数据",
+
+  "data_tablet.tabs.basics.name": "基础信息",
+  "data_tablet.tabs.essence_manipulation.name": "源质操纵",
+  "data_tablet.tabs.machinery.name": "机器与加工",
+  "data_tablet.tabs.machinery_transformation.name": "机器与转化",
+  "data_tablet.tabs.machinery_cultivation.name": "机器与培育",
+  "data_tablet.tabs.machinery_ascension.name": "机器与飞升",
+  "data_tablet.tabs.tools.name": "工具与实用物品",
+  "data_tablet.tabs.lore.name": "古代历史",
+
+  "_comment.data_tablet_pages": "数据平板页面文本。",
+  "_comment.data_tablet_pages.basics": "基础信息分类文本。",
+  "data_tablet.pages.components.name": "基础合成零件",
+  "data_tablet.pages.components.page1.text": "不加精制的材料有效用上限，所以我们设计了许多更精细的合成用零件，可大大简化源质动力机器的组装流程。有些零件只是机械零件，有些带有电路，还有些更加高级。",
+  "data_tablet.pages.data_tablet.flavor": " ◇ 古代典籍",
+  "data_tablet.pages.data_tablet.name": "数据平板",
+  "data_tablet.pages.data_tablet.page1.text": "每位研究者都会获得一块个人数据平板，用于记录我们的发现、发明、知识，也即数据。前往任意一所活跃的研究中心，如有许可，即能从那里的数据库复制并更新平板的内容。\n\n数据平板拥有一系列安全保密机制，用于阻止低级别平板接受高级别数据。研究员在获取当前级别的所有关键数据，并取得来自高级别同行的许可后，可将平板晋级。这是为确保我们积累的知识不落入坏人之手，也是为检验研究员是否有足够能力接受我们的教导。\n\n关键数据载入数据平板的过程通常分为两步。第一部分一般是一系列提示，或是不完整的研究日志。在沿着我们的足迹探索之后，第二部分，也就是完整的数据条目，会覆盖第一部分。完成条目后，平板会将该关键数据登记为已解锁，并可能会开放某所研究中心内进一步推进的权限。",
+  "data_tablet.pages.data_tablet.page2.text": "数据平板的界面分为多个标签页，标签页下为对应的条目。条目之间通常会存在先后关系；如果双方位于同一标签页下，则两者间会以发光的线标识。\n\n查看条目时，可使用右侧的书与笔按钮“保存”当前页；再次打开平板时会直接跳转至该页，而非进入主界面。潜行点击按钮可清除保存。\n\n如果某条目的文本超出了屏幕长度，即会出现一个滚动条，方便查看完整文本。",
+  "data_tablet.pages.data_tablet.page3.text": "平板能够显示文本、栩栩如生的方块与物品图例、多种配方图标，以及多方块结构蓝图。结构蓝图可以通过内置的投影器投射到世界中预览，按下多方块页面的眼睛图标即可。",
+  "data_tablet.pages.essence_crystals.name": "源质晶体",
+  "data_tablet.pages.essence_crystals.page1.text": "洞穴中会存在原生的源质晶体矿簇。截至数据上传时，它们的来源仍未被解明，但我们推测它们的形成曾经是一种自然过程。无论如何，挖掘晶体会导致它碎裂成多片较小的碎片，这一点在刚开始利用源质时有着莫大的帮助。",
+  "data_tablet.pages.essence_crystals.page2.text": "已有实证表明，紫水晶可在灌注器中转变成源质碎片，供应不足时可以应急。此过程消耗的源质并不会少于单片源质碎片的产出量，不然任何新人研究员都该大量使用了。",

```

```suggestion
  "data_tablet.pages.essence_crystals.page2.text": "已有实证表明，紫水晶可在灌注器中转变成源质碎片，供应不足时可以应急。以防有些新人动歪主意，还需申明，此过程消耗的源质并不少于单片源质碎片的产出量。",
```
This procedure does not consume less Essence than a Shard produces, *lest (in case) any novice researchers get any ideas*.
这是说「消耗比生成少那不就永动机了吗」吧（

---

我也是这么觉得的，没翻出来（